new/LIT7047Qene.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7047Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ba-zaman qadāmāwi ʾi-yǝḥǝmmi qʷāʿ</title>
+                <title xml:lang="en" corresp="#t1">In the old times the raven was not jealous</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"></witness>
+                </listWit> </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-08-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rb.</note>
+                <ab>
+                    <l n="1">በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡</l>
```

Comment:
If I understand correctly this is a qene of wāzemā type, with five verses, the first one being very long; 
a bit strange, but I saw already qenes with the 1st line disproportionally long. The division into verses is indicated with ።
Then we have as the first verse: በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡ ኤልያስሃ፡ በተአዝዘቱ፡ አክራመ፡ ፫፡ እንበለ፡ መብልዕ፡ ወልብስ፡ ለዕንቋዕ፡ ቋዕ፡ ዓቅሙ፡ ንዑስ።